### PR TITLE
feat: generate keys for system data contracts

### PR DESCRIPTION
**Pull request by ktechmidas**
## Issue being fixed or feature implemented

We now generate HD keys using wallet-lib instead of the old way using Dashmate, this allows us to remove some dependencies and add 

## What was done?

* Removed individual dependencies
* Added dependency for wallet-lib
* Removed old keys from ansible generation file
* Added new HD keys
* Added new HD keys to docker-compose for drive

## How Has This Been Tested?

Deployed devnet-krupnik with this PR enabled

## Breaking Changes

This will likely break any older devnets including malort, these devnets either need to be updated or regenerated. However some devnets do not require platform to be working.

## Checklist:

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added or updated relevant unit/integration/functional/e2e tests
- [ ] I have made corresponding changes to the documentation

**For repository code-owners and collaborators only**
- [ ] I have assigned this pull request to a milestone

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -24,6 +24,11 @@
     "@dashevo/dapi-client": "~0.21.2",
     "@dashevo/dashcore-lib": "~0.19.26",
     "@dashevo/dashd-rpc": "^2.3.1",
+    "@dashevo/dashpay-contract": "^0.21.7",
+    "@dashevo/dpns-contract": "^0.21.7",
+    "@dashevo/feature-flags-contract": "^0.21.7",
+    "@dashevo/masternode-reward-shares-contract": "^0.22.0-dev.5",
+    "@dashevo/wallet-lib": "~7.21.7",
```
Comment (shumkov) [inline package.json:31]: You better use `0.22.0-dev.7`.

File: package.json [hunk 2]
```diff
@@ -24,6 +24,11 @@
     "@dashevo/dapi-client": "~0.21.2",
```
Comment (shumkov) [inline package.json:24]: You better use `0.22.0-dev.7`.

**Review by shumkov — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (ktechmidas) [reply package.json:31]: Got it, not sure why it used 0.21...

File: lib/configGenerator/generateAnsibleConfig.js [hunk 3]
```diff
@@ -1,13 +1,43 @@
...
 const { PrivateKey } = require('@dashevo/dashcore-lib');
+const { Wallet } = require('@dashevo/wallet-lib');
 
 const BlsSignatures = require('bls-signatures');
 
 const yaml = require('js-yaml');
 
 const crypto = require('crypto');
 
+const { contractId: dpnsContractId, ownerId: dpnsOnwerId } = require('@dashevo/dpns-contract/lib/systemIds');
```
Comment (shumkov) [inline lib/configGenerator/generateAnsibleConfig.js:12]: You don't need this IDs and dependencies what you added in package.json anymore.

Comment (ktechmidas) [reply lib/configGenerator/generateAnsibleConfig.js:12]: Huh, Djavid went through with me and added these... it wasn't there before the PR!

File: lib/configGenerator/generateAnsibleConfig.js [hunk 4]
```diff
@@ -19,15 +44,19 @@ async function generateAnsibleConfig(network, networkName, masternodesCount, see
...
-  config.platform_feature_flags_contract_block_height = '';
-  config.platform_feature_flags_top_level_identity = '';
+  config.dpns_hdprivkey = '';
+  config.dpns_hdpubkey = '';
+  config.dashpay_hdprivkey = '';
+  config.dashpay_hdpubkey = '';
+  config.ff_hdpubkey = '';
+  config.ff_hdprivkey = '';
+  config.mnrewardshares_hdpubkey = '';
+  config.mnrewardshares_hdprivkey = '';
```
Comment (shumkov) [inline lib/configGenerator/generateAnsibleConfig.js:54]: i think we don't need this

**Review by shumkov — CHANGES_REQUESTED** [review]

**Review by shumkov — APPROVED** [review]
LGTM! 👍

**Pull request merged**